deployments/deploy-tests.ps1

```diff
@@ -142,24 +143,70 @@ try{
     #
     #############################################################################################################
 
-    #Still need to write the get function, but the thought is that you could pull a resource just by defining
+    #The thought is that you could pull a resource just by defining
     #the name and the context will be used to get the rg and sub to pull it from
  
-    #Set-AzCloudContext -SubscriptionName "MYAEA-KKZH-D" -ResourceGroupName "RG-MYAEA-KKZH-ADM-D"
-    #$kv = Get-KeyVaultDefinition -Name "KV-MYAEA-KKZH-ADM-C1-D01"
-
+    Set-PfAzureContext -SubscriptionName "MYAEA-KKZH-D" -ResourceGroupName "RG-MYAEA-KKZH-ADM-D" -AzRegion $([AzRegions]::CentralUs)
+    $kv = Get-PfKeyVaultCloudState -Name "KV-MYAEA-KKZH-ADM-C1-D01"
+    $kv
 
     ##############################################################################################################
     #
     #               Test Loading a single resource only
     #
     #############################################################################################################
 
-    #Still need to write the get function, but the thought is that you could pull a resource without any 
+    #The thought is that you could pull a resource without any 
     #az context or deployment context.
     #And if you even had a context set, this would still load that resource from any rg or sub outside the context
     
-    #$kv = Get-KeyVaultDefinition -SubscriptionName "MYAEA-KKZH-D" -ResourceGroupName "RG-MYAEA-KKZH-ADM-D" -Name "KV-MYAEA-KKZH-ADM-C1-D01"
+    $kv = Get-PfKeyVaultCloudState -SubscriptionName "MYAEA-KKZH-D" -ResourceGroupName "RG-MYAEA-KKZH-ADM-D" -Name "KV-MYAEA-KKZH-ADM-C1-D01"
+    $kv
+
+     ##############################################################################################################
+    #
+    #               Get resources with context set with "enterprise standards"
+    #
+    #############################################################################################################
+
+    #Shows how you can use the get cloud state function with enterprise standards defined in context.
+    #Also shows how you can get a resource within a deployment context as well
+
+    New-PfDeploymentContext
+
+    Set-PfAzureContext -CompanyAbbreviation "MYAEA" -GroupAbbreviation "KKZH" -Label "ADM" -EnvironmentLetter "D" -AzRegion $([AzRegions]::CentralUs)
+
+    $kv = Get-PfKeyVaultCloudState
+    $kv
+
+    #name, az context built from deployment context params above
+    $kv2 = New-PfKeyVaultBuild
+    $kv2.Options.Name = "testing"
+    
```

Comment:
Wondering about possibility to  drop off suffixes, if we not going to have variants of Get-PfKeyVault* and New-PfKeyVault* 

Can we new up or get other types of 'objects' reprsenting a KV?

Reply:
This is a good point and I like the idea.  I can't think of other objects, so I will move forward with this until we hit a roadblock if any.

Reply:
This is done